blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.jsx

```diff
@@ -25,44 +27,46 @@ const RightRailAdvancedLayout = ({ children }) => {
 		rightRailBottom,
 		fullWidth2,
 		footer,
-	] = children;
+	] = React.Children.toArray(children);
 	const featureList = useFeatureList();
 
 	return (
-		<>
-			<header className="page-header">{navigation}</header>
-			<section role="main" id="main" className="main" tabIndex="-1">
-				<div className="container layout-section">
-					<div className="row">
-						<div className="col-sm-xl-12 layout-section wrap-bottom">{fullwidth1}</div>
-					</div>
-					<div className="row advanced-grid">
-						<section className="col-sm-md-12 col-lg-xl-8 advanced-grid-desktop-main-area">
-							<RenderChild Item={main} tabletPlacement="2" />
-							<RenderChild Item={main2} tabletPlacement="4" />
-						</section>
-						<section className="col-sm-md-12 col-lg-xl-4 layout-section">
-							<RenderChild Item={rightRailTop} tabletPlacement="1" />
-							<RenderChild Item={rightRailMiddle} tabletPlacement="3" />
-							<RenderChild Item={rightRailBottom} tabletPlacement="5" />
-						</section>
-					</div>
-					{featureList["7"] > 0 && (
-						<div className="row">
-							<div className="col-sm-xl-12 layout-section wrap-bottom">{fullWidth2}</div>
-						</div>
-					)}
-				</div>
+		<div className={LAYOUT_CLASS_NAME}>
+			{navigation ? (
+				<Stack role="banner" className={`${LAYOUT_CLASS_NAME}__navigation`}>
+					{navigation}
+				</Stack>
+			) : null}
+
+			<section role="main" tabIndex="-1" className={`${LAYOUT_CLASS_NAME}__main`}>
+				{fullwidth1 ? (
+					<Stack className={`${LAYOUT_CLASS_NAME}__full-width-1`}>{fullwidth1}</Stack>
+				) : null}
+
+				<Grid className={`${LAYOUT_CLASS_NAME}__rail-container`}>
+					<Stack className={`${LAYOUT_CLASS_NAME}__main-interior-item`}>
+						<Stack className={`${LAYOUT_CLASS_NAME}__main-interior-item-1`}>{main}</Stack>
+						<Stack className={`${LAYOUT_CLASS_NAME}__main-interior-item-2`}>{main2}</Stack>
+					</Stack>
+					<Stack role="complementary" className={`${LAYOUT_CLASS_NAME}__main-side-rail`}>
```

Comment:
`role="complementary` is not needed here

Reply:
ok will remove. saw "aside" element was used before

Reply:
removed